.vscode/settings.json

```diff
@@ -2,8 +2,8 @@
   "editor.tabSize": 2,
   "editor.formatOnType": true,
   "google-java-format.executable": "/usr/local/bin/google-java-format",
-  "editor.formatOnPaste": false,
-  "editor.formatOnSave": false,
+  "editor.formatOnPaste": true,
+  "editor.formatOnSave": true,
```

Comment:
These were left as false by mistake in a pervious merge. The IDE _should_ automatically format on save.